packages/entity-server/src/routes/utils/ResponseObjectBuilder.ts

```diff
@@ -0,0 +1,18 @@
+/**
+ * Tupaia
+ * Copyright (c) 2017 - 2021 Beyond Essential Systems Pty Ltd
+ */
+
+import { Writable } from '../../types';
+
+export class ResponseObjectBuilder<T> {
```

Comment:
I extracted this because I thought I'd reuse it in the new endpoint.

Turns out I didn't have too, but I think it still makes sense to be a generic class.